<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigSub</class>
 <widget class="QWidget" name="ConfigSub">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConfigSub</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="filePathLabel">
     <property name="text">
      <string>No file selected</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="browseButton">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="processButton">
     <property name="text">
      <string>Process File</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
